lib/sa.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, Page } from "@playwright/test";
+import { config } from "./config";
+
+export const navigateToSAList = async function (page: Page) {
+    await page.getByRole('link', { name: 'Application and Data Services' }).click();
+    // }
+    await page.getByRole('link', { name: 'Service Accounts' }).click();
+    await expect(page.getByRole('heading', { name: 'Service Accounts' })).toHaveCount(1);
+};
+
+export const createServiceAccount = async function (page: Page, name: string) {
+    await page.locator('button', { hasText: 'Create service account' }).click();
+
+    await expect(page.getByText('Create a service account', {exact: true})).toHaveCount(1);
+
+    await page.waitForSelector('[role=progressbar]', { state: 'detached' });
+
+    await page.getByLabel('Short description *').fill(name);
+
+    await page.getByTestId('modalCreateServiceAccount-buttonSubmit').click();
```

Comment:
@Frawless I know you and @kornys removed probably all data-testid locators, I tried to find another reliable locator for this one, but no other except maybe for `await page.getByText('Create', {exact: true}).click();` would work.
If I used the simple `page.locator('button', { hasText: 'Create' })`, there is no exact match option, so it returns even the 'Create service account' button hidden behind the modal.

Reply:
can you try `page.locator('.button :text-is("Create")')` ?

Reply:
Strangely enough, removing the dot before the button and the space after it (`'button:text-is("Create")'`) did the trick. The space is in the documentation, so I don't really get it.
Or perhaps using `.getByText` with regular expressions  `'/^Create$/'` could do.

Reply:
Interesting...I am not UI testing guru so I will let decision up to you :)

Reply:
I think as long as the tests will be stable we are fine

Reply:
is the commit missing?